classes/classes/lists/GenitalLists.as

```diff
@@ -18,7 +18,7 @@ package classes.lists
 			CockTypesEnum.FOX,
 			CockTypesEnum.WOLF,
 			CockTypesEnum.DRAGON,
-			CockTypesEnum.DISPLACER,
+			CockTypesEnum.DISPLACER
```

Comment:
A trailing comma in an array is not a mistake. Its common practice and intended.
Its to keep diffs less noisy, when adding more items to the list.

Reply:
Not saying it's a mistake, just that the parser cannot deal with it.